arch/arm/src/stm32/hardware/stm32_dma_v1.h

```diff
@@ -54,6 +54,10 @@
 #define DMA1                       (0)
 #define DMA2                       (1)
 
+#if defined(CONFIG_STM32_STM32G47XX)
+#  define HAVE_DMA_CHAN8
+#endif
```

Comment:
This should come in from chip.h or Kconfig.

Reply:
> This should come in from chip.h or Kconfig.

I think there should there be **two** defines from Kconfig: HAVE_DMA1_CHAN8 and HAVE_DMA2_CHAN678.

Reply:
Done in b5910332e199ae2d9aeabbe1e1341198f43f7b9c.